### PR TITLE
Add description and disabled to logging sinks

### PR DESCRIPTION
**Pull request by ScottSuarez**
Upstreaming [7795](https://github.com/hashicorp/terraform-provider-google/pull/7795/files)

If this PR is for Terraform, I acknowledge that I have:

- [x] Searched through the [issue tracker](https://github.com/hashicorp/terraform-provider-google/issues) for an open issue that this either resolves or contributes to, commented on it to claim it, and written "fixes {url}" or "part of {url}" in this PR description. If there were no relevant open issues, I opened one and commented that I would like to work on it (not necessary for very small changes).
- [x] [Generated Terraform](https://github.com/GoogleCloudPlatform/magic-modules#generating-downstream-tools), and ran `make test` and `make lint` to ensure it passes unit and linter tests.
- [x] Ensured that all new fields I added that can be set by a user appear in at least one [example](https://github.com/GoogleCloudPlatform/magic-modules/tree/master/templates/terraform/examples) (for generated resources) or [third_party test](https://github.com/GoogleCloudPlatform/magic-modules/tree/master/third_party/terraform/tests) (for handwritten resources or update tests).
- [x] [Ran](https://github.com/hashicorp/terraform-provider-google/blob/master/.github/CONTRIBUTING.md#tests) relevant acceptance tests (If the acceptance tests do not yet pass or you are unable to run them, please let your reviewer know).
- [x] Read the [Release Notes Guide](https://github.com/GoogleCloudPlatform/magic-modules/blob/master/.ci/RELEASE_NOTES_GUIDE.md) before writing my release note below.

**Release Note Template for Downstream PRs (will be copied)**

```release-note:enhancement
logging: added `description` field to `google_logging_folder_sink` resource
logging: added `description` field to `google_logging_project_sink` resource
logging: added `description` field to `google_logging_organization_sink` resource
logging: added `description` field to `google_logging_billing_account_sink` resource
logging: added `disabled` field to `google_logging_folder_sink` resource
logging: added `disabled` field to `google_logging_project_sink` resource
logging: added `disabled` field to `google_logging_organization_sink` resource
logging: added `disabled` field to `google_logging_billing_account_sink` resource
```

### CODE REVIEW

**Comment by dabbertorres** [comment]
@googlebot I fixed it.

**Comment by dabbertorres** [comment]
@googlebot I consent.

File: third_party/terraform/tests/resource_logging_billing_account_sink_test.go [hunk 1]
```diff
@@ -221,6 +279,36 @@ resource "google_storage_bucket" "log-bucket" {
 `, name, billingAccount, getTestProjectFromEnv(), bucketName)
 }
 
+func testAccLoggingBillingAccountSink_described(name, bucketName, billingAccount string) string {
```
Comment (slevenick) [inline third_party/terraform/tests/resource_logging_billing_account_sink_test.go:282]: I think the `description` field was left out of this config

File: third_party/terraform/tests/resource_logging_billing_account_sink_test.go [hunk 2]
```diff
@@ -221,6 +279,36 @@ resource "google_storage_bucket" "log-bucket" {
...
+  filter          = "logName=\"projects/%s/logs/compute.googleapis.com%%2Factivity_log\" AND severity>=ERROR"
+}
+
+resource "google_storage_bucket" "log-bucket" {
+  name = "%s"
+}
+`, name, billingAccount, getTestProjectFromEnv(), bucketName)
+}
+
+func testAccLoggingBillingAccountSink_disabled(name, bucketName, billingAccount string) string {
```
Comment (slevenick) [inline third_party/terraform/tests/resource_logging_billing_account_sink_test.go:297]: Similar, `disabled` was left out of this config

**Review by slevenick — APPROVED** [review]
Looks reasonable. A couple of the tests don't have the fields they are supposed to be testing

File: third_party/terraform/tests/resource_logging_billing_account_sink_test.go (hunk 2 above)
Comment (ScottSuarez) [reply third_party/terraform/tests/resource_logging_billing_account_sink_test.go:290]: done

**Pull request merged**